### DISCUSSION TITLE
Plugged in wrong way

### DISCUSSION THREAD

**Discussion by ziogref** [revk/xESP32-Faikout#201] [General]
I installed my faikin (long story) backwards and I think I fried it. If I feed it 5v and ground will it turn on? or do I need the 12v connected aswell?

**Comment by revk** [comment]
It is powered from the 12V. The 5V is optional and just used as a pull up on Tx line.

You need `GND` and `12V` connected.

**Reply by ziogref** [reply]
Thanks, Tested ok
Looks like the Unit needed be power-cycled and now it all works again

Thanks for your prompt response
